src/community/authkey/src/main/java/org/geoserver/security/GeoServerAuthenticationKeyFilter.java

```diff
@@ -24,99 +23,95 @@
 import org.geoserver.security.config.SecurityNamedServiceConfig;
 import org.geoserver.security.filter.AuthenticationCachingFilter;
 import org.geoserver.security.filter.GeoServerAuthenticationFilter;
-import org.geoserver.security.filter.GeoServerSecurityContextPersistenceFilter;
 import org.geoserver.security.filter.GeoServerSecurityFilter;
 import org.geoserver.security.impl.GeoServerRole;
 import org.geoserver.security.impl.GeoServerUser;
+import org.geoserver.security.impl.GeoServerUserGroup;
+import org.geoserver.security.impl.RoleCalculator;
 import org.springframework.security.core.Authentication;
 import org.springframework.security.core.GrantedAuthority;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.security.web.AuthenticationEntryPoint;
 import org.springframework.security.web.authentication.Http403ForbiddenEntryPoint;
 import org.springframework.util.StringUtils;
 
-
 /**
  * Filter extending {@link GeoServerSecurityFilter}.
  * 
- *  The encoded user name is passed as an URL parameter named {@link #authKeyParamName}.
- *  
- *  The real user name is retrieved by querying an {@link AuthenticationKeyMapper} object stored
- *  in {@link #authKeyMapperName}
- *  
- *  This filter needs a {@link GeoServerUserGroupService} for authentication
+ * The encoded user name is passed as an URL parameter named {@link #authKeyParamName}.
+ * 
+ * The real user name is retrieved by querying an {@link AuthenticationKeyMapper} object stored in {@link #authKeyMapperName}
+ * 
+ * This filter needs a {@link GeoServerUserGroupService} for authentication
  * 
  * @author christian
  *
  */
-public class GeoServerAuthenticationKeyFilter extends  GeoServerSecurityFilter 
-    implements AuthenticationCachingFilter, GeoServerAuthenticationFilter  
-    {
-    
-    private String authKeyMapperName,authKeyParamName;
+public class GeoServerAuthenticationKeyFilter extends GeoServerSecurityFilter
```

Comment:
Is there anything changed in this file? I can see many reformats, but it is not clear if any line of code was really changed

Reply:
None, just reformat accordingly to geoserver code conventions.